libs/designer/src/lib/ui/mcp/operations/SelectOperations.tsx

```diff
@@ -31,8 +45,20 @@ export const SelectOperations = () => {
   const selectedConnector = useMemo(() => allConnectors?.find((c) => c.id === selectedConnectorId), [allConnectors, selectedConnectorId]);
 
   const operations = useMemo(() => {
-    return allOperations || [];
-  }, [allOperations]);
+    const allOps = allOperations || [];
+
+    const trimmedSearchTerm = searchTerm.trim();
+
+    // If no search keyword, return all operations sorted alphabetically
+    if (isUndefinedOrEmptyString(trimmedSearchTerm)) {
+      return [...allOps].sort((a, b) => (a.name || '').localeCompare(b.name || ''));
```

Comment:
Shouldn't you be sorting based on the display names? What is the sort algorithm in designer today..
because all you see is the display names so we should be using that